pkg/util/hostname/common.go

```diff
@@ -34,10 +35,7 @@ var (
 )
 
 // Data contains hostname and the hostname provider
-type Data struct {
-	Hostname string
-	Provider string
-}
+type Data = hostnameinterface.Data
```

Comment:
Is there a reason to introduce a type alias as `hostname.Data` is not used at a lot of locations?

Reply:
We're introducing aliases throughout our module refactors in order to keep the diffs smaller for reviews, and have a ticket in our backlog to remove the aliases after the refactor is done.